src/main/java/com/iridium/iridiumskyblock/database/Island.java

```diff
@@ -112,6 +115,7 @@ public int getLevel() {
      * @return The experience required to reach this level
      */
     private int getExperienceRequired(int level) {
+        //TODO
         return -1 + (level * level * level);
```

Comment:
Maybe we can create an equation like `level=cbrt(x)` where level is defined, and we can just solve for x? I don't know if Crunch can do that, though.